tools/report-converter/tests/unit/smatch_output_test_files/files/sample.c

```diff
@@ -0,0 +1,5 @@
+static struct parse_smatch *check_sample(){
+			int res;
+			res = 0;
+			return ERR_PTR(res);
+		}
```

Comment:
Maybe we can format this file a little bit:
```suggestion
static struct parse_smatch *check_sample() {
  int res;
	res = 0;
	return ERR_PTR(res);
}
```

Reply:
I am so sorry, I forgot about formating the sample file :sweat_smile: I will format it and rebase with the test commit